terraform/modules/rds_migration/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "db_migration_ecs_security_group_id" {

```

Comment:
Is this being used anywhere?

Reply:
Nice spot! It isn't, it's a leftover from some implementation I was trying. Will remove.